etc/config/c++.amazon.properties

```diff
@@ -320,6 +320,10 @@ compiler.clang1400.exe=/opt/compiler-explorer/clang-14.0.0/bin/clang++
 compiler.clang1400.semver=14.0.0
 compiler.clang1400.options=--gcc-toolchain=/opt/compiler-explorer/gcc-11.2.0
 compiler.clang1400.ldPath=${exePath}/../lib|${exePath}/../lib/x86_64-unknown-linux-gnu
+compiler.clang1500.exe=/opt/compiler-explorer/clang-15.0.0/bin/clang++
+compiler.clang1500.semver=15.0.0
+compiler.clang1500.options=--gcc-toolchain=/opt/compiler-explorer/gcc-11.2.0
```

Comment:
You can probably point to gcc-12.2.0 here

Reply:
Done